### PR TITLE
update empty dashboard message

### LINKED ISSUE #141: update empty dashboard message
When the user first creates an account and lands on an empty dashboard, it's not clear to them what to do next. We should add a link to browse courses. 

#### Designs and Mockups

https://projects.invisionapp.com/d/main/#/console/21535972/455982830/inspect

#### Acceptance Criteria:

- [ ] Change the empty dashboard message from `Once you enroll in a course, you can find it listed here.` to `You are not enrolled in any courses yet. Please browse our courses.` with `browse our courses` a link to the home page. We can change it to the catalog page when we have a catalog page.

### PR DESCRIPTION
**Pull request by umarmughal824**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
fixes #141 

#### What's this PR do?
update empty dashboard message

#### How should this be manually tested?
Visit the dashboard page

#### Screenshots (if appropriate)

**NOW**

Screenshot 2021-09-02 at 18 50 54

**BEFORE**

Screenshot 2021-09-02 at 13 34 31

### CODE REVIEW

**Review by pdpinch — APPROVED** [review]
screenshots look good to me

File: static/js/containers/pages/DashboardPage.js
Comment (gsidebo) [inline static/js/containers/pages/DashboardPage.js:83]: Your `.placeholder-card` styles just copy the styles for `.card` in `common.scss`. Can you get rid of those styles and change this approach to avoid the repetition?

```html
<div className="card no-enrollments p-3 p-sm-5 rounded-0">
  ...
</div>
```

```scss
// In dashboard.scss...

.enrolled-items {
  .card {
    display: flex;
    flex-direction: row;
    
    &.no-enrollments {
      flex-direction: column;
      
      a {
        color: black;
        text-decoration: underline;
      }
    }
    
```

**Review by gsidebo — CHANGES_REQUESTED** [review]

File: static/scss/dashboard.scss
Comment (gsidebo) [inline static/scss/dashboard.scss:18]: The `<p>` tag is adding extra space to the bottom of this card. Can you add a rule to remove that margin under the `&.no-enrollments` selector?

```suggestion
    &.no-enrollments {
      flex-direction: column;

      a {
        color: black;
        text-decoration: underline;
      }
      
      p {
        margin: 0;
      }
    }
```

**Review by gsidebo — APPROVED** [review]
Looks great! I just had 1 comment about adding one more styling rule. Once you add that, feel free to merge right away 👍

**Pull request merged**
